pkg/machines/libvirt-dbus.es6

```diff
@@ -654,6 +655,30 @@ LIBVIRT_DBUS_PROVIDER = {
         };
     },
 
+    SET_MEMORY ({
+        name,
+        id: objPath,
+        connectionName,
+        memory
+    }) {
+        const updatedMemory = memory * 1024;
+        return call(connectionName, objPath, 'org.libvirt.Domain', 'SetMemory', [updatedMemory, 0], TIMEOUT);
+    },
+
+    SET_MAX_MEMORY ({
+        name,
```

Comment:
You don't use the name anywhere, you can remove it from here and from the action as well.